airbyte-webapp/src/components/ui/NumberBadge/NumberBadge.module.scss

```diff
@@ -17,23 +17,21 @@
   font-style: normal;
   font-weight: 500;
   font-size: 10px;
+  color: colors.$white;
```

Comment:
This is the wrong text color for this background color. It's not fulfilling WCAG AA contrast ratios. The supposed text color on those background colors were correct before this PR.

Reply:
This was not an arbitrary change, but the current design has not been updated to reflect this. Neither @Upmitt nor I were aware that this change was made for this reason. I have reverted the change.